crates/swc_css_parser/src/parser/util.rs

```diff
@@ -14,29 +12,6 @@ impl<I> Parser<I>
 where
     I: ParserInput,
 {
-    // TODO: error recovery.
-    pub(super) fn parse_delimited<T>(&mut self, allow_zero: bool) -> PResult<Vec<T>>
-    where
-        Self: Parse<T> + ParseDelmited<T>,
-    {
-        let mut items = vec![];
-
-        if allow_zero && !ParseDelmited::eat_delimiter(self)? {
-            return Ok(vec![]);
-        }
-
-        loop {
-            let res = self.parse()?;
-            items.push(res);
-
-            if !ParseDelmited::eat_delimiter(self)? {
-                break;
-            }
-        }
-
-        Ok(items)
-    }
```

Comment:
In future (when we refactor parser using grammars in right way - we are working on it) we will have the same util, but it will be refactor to using `ComponentValue`, currently we don't need it